librz/core/canalysis.c

```diff
@@ -7262,6 +7265,50 @@ RZ_IPI char *rz_core_analysis_all_vars_display(RzCore *core, RzAnalysisFunction
 	return rz_strbuf_drain(sb);
 }
 
+static int check_rom_exists(const void *value, const void *data) {
+	const char *name = (const char *)value;
+	const RzBinSection *sections = (const RzBinSection *)data;
+	return strcmp(name, sections->name);
+}
+
+/**
+ * \brief Gets the ROM_ADDRESS and ROM_SIZE from the corresponding CPU Profile
```

Comment:
This is not a brief. Please split it into one-line `brief` and more detailed explanation below